components/core/routes/Download.py

```diff
@@ -33,7 +33,8 @@ def kill():
 			p.join()
 			jsonreq = json.dumps({'jsonrpc': '2.0', 'id': 'qwer', 'method': 'aria2.pauseAll'})
 			jsonreq = jsonreq.encode('ascii')
-			c = urllib.request.urlopen('http://localhost:6800/jsonrpc', jsonreq)
+			# c = urllib.request.urlopen('http://localhost:6800/jsonrpc', jsonreq)
```

Comment:
You can remove the unnecessary code instead of commenting it out :)

Reply:
Sure :)